app/src/main/java/com/sdpteam/connectout/event/EventBuilder.java

```diff
@@ -0,0 +1,62 @@
+package com.sdpteam.connectout.event;
+
+import com.google.android.gms.maps.model.LatLng;
+
+public class EventBuilder {
```

Comment:
I agree, that in the context of this issue, this class is not really useful. 
However, by separating the construction of Event objects into a separate class, the EventBuilder can help to keep the Event class itself simple and focused (KISS). the use of this builder also helps to provide immutability in the class it builds.
This might be relevant for managing inputs from firebase.